<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bill</class>
 <widget class="QMainWindow" name="bill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BILLING MODULE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/main.png</normaloff>icons/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BILLING MODULE</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CUSTOMER NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="contact">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CUSTOMER CONTACT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ccontact">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="contact_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DATE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="contact_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TIME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="contact_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INVOICE NUMBER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="billno">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Item Quantity</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="iqt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Kozuka Mincho Pro B</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RESET ALL</string>
    </property>
   </widget>
   <widget class="QComboBox" name="icategory">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="iname">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Remarks: </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="remarks">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tnc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>T &amp; C *</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Discount/Scheme: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="discount">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="name_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ISSUED BY NAME / ID </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="issuedby">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addtocart">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>ADD TO CART</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>TOTAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generatebill">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rod</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>GENERATE BILL</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>361</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro Cond</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CART</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro Bold</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>REMOVE</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>icategory</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>iname</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>iqt</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
